src/wl/sys/wl_cfg80211_hybrid.c

```diff
@@ -55,13 +55,22 @@ u32 wl_dbg_level = WL_DBG_ERR | WL_DBG_INFO;
 u32 wl_dbg_level = WL_DBG_ERR;
 #endif
 
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(6, 1, 0)
+static int wl_cfg80211_change_iface(struct wiphy *wiphy,
```

Comment:
I'd prefer if we could stick with `s32` instead of `int` to keep the code maintainable and patches easy to cherry pick.

Reply:
Does that work? It looks like `s32` for these functions has been changed to `int` all across 6.0-rc1...

Reply:
It's simply a matter of style as far as I can tell.

The rationale is that this codebase is in maintenance mode, and I'm already violating its license by sharing a modified version of it. Therefore I tend to avoid pushing changes which are purely related to code style, small refactoring, etc. (even if that style is used upstream), and patch function signatures here and there to ensure it still builds as the Linux kernel evolves.

Those patch may be ugly as hell, but they ensure the least amount of changes to Broadcom's intellectual property.